backend/src/routes/index.js

```diff
@@ -1,12 +1,14 @@
-import express from "express";
-import serverRoute from "./serverRoute.js";
-import notificationRoute from "./notificationRoute.js";
-import userRoute from "./userRoute.js";
+import express from 'express';
+import serverRoute from './serverRoute.js';
+import notificationRoute from './notificationRoute.js';
+import userRoute from './userRoute.js';
+import authenticate from '../middleware/application/authenticate.js';
+
 
 const router = express.Router();
 
 router.use("/auth", userRoute);
-router.use("/server", serverRoute);
-router.use("/server", notificationRoute);
+router.use("/server", authenticate, serverRoute);
+router.use("/server", authenticate, notificationRoute);
```

Comment:
You shouldn't place your middlewares here. They should be   in the notificationRoute or serverRoute files